onboarder/installation/index.rst

```diff
@@ -44,6 +44,15 @@ Import the extension configuration in the ``app/config/config.yml`` file (after
     imports:
         - { resource: '@PimOnboarderBundle/Resources/config/onboarder_configuration.yml' }
 
+Build the front
```

Comment:
Maybe we say "Build the UI". "front" can mean many things. At Akeneo we use it to designate the UI, but it could also refer to a web server for instance.

Reply:
Not mandatory, of course, just wondering.

Reply:
The more accurate, the better :+1: